src/background.ts

```diff
@@ -245,6 +245,9 @@ omnibox.init()
 
 // }}}
 
+
+setTimeout(config.update, 5000)
```

Comment:
This is clearly a bad idea, but it fixes things in my testing. We must have some race conditions.

Reply:
@glacambre I'm probably going to merge this tomorrow unless you tell me it's too terrible an idea. I don't really have the time to chase down the race conditions and people are hitting the sync quota already.

The downside is that Tridactyl will have a weird configuration for a few seconds (in this case, the default configuration but it later versions it will be old configuration) until this updater kicks in.